src/lib/peer.ts

```diff
@@ -28,7 +28,7 @@ export class PeerAPI extends BaseAPI {
 
     this.cache[address] = this.lambdasClient
       .fetchProfiles([address.toLowerCase()], undefined, {
-        headers: { 'Cache-Control': 'max-age=0', Pragma: 'no-cache' }
+        headers: { 'Cache-Control': 'max-age=0' }
```

Comment:
FYI: The catalyst are not supporting the `Pragma` header 😢 so I'm removing it for now.
![image](https://user-images.githubusercontent.com/8763687/210804935-26b5c4be-6252-44cb-958c-7a14cc3e6b61.png)

Reply:
@menduz FYI